src/main/scala/tile/RocketTile.scala

```diff
@@ -137,6 +137,14 @@ class RocketTile(
     }
   }
 
+  class RocketLogicalTree extends LogicalTree {
+    override def getOMComponents(resourceBindingsMap: ResourceBindingsMap, components: Seq[OMComponent]): Seq[OMComponent] = {
```

Comment:
Why is `components` an argument here?

Reply:
Child components are the OM children of the current node and are added to the current node.

  def tree(t: Tree[LogicalTree], resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] = {
    val childComponents =  t.children.flatMap(tree(_, resourceBindingsMap))
    t.parent.getOMComponents(resourceBindingsMap, childComponents)
  }